src/web/mod.rs

```diff
@@ -571,16 +634,18 @@ mod roles {
         ))
     }
 
-    pub fn service() -> Scope {
+    pub fn scope() -> Scope {
         web::scope("/roles").service(index)
     }
 }
 
+#[allow(clippy::unused_async)]
 async fn default_route() -> impl Responder {
     HttpResponse::NotFound().body("Not Found")
 }
 
 #[get("")]
+#[allow(clippy::unused_async)]
```

Comment:
Consider fixing. Async calls have a cost, and aren't always good.

Reply:
`actix_web` requires its "services" to by async functions.

Reply:
I would open a ticket to make at least one of the contained functions `async`, so the `unused_async` lint passes.

Reply:
Where you'd open a ticket? In the Actix repo?